highcharts/src/main/java/us/ascendtech/highcharts/client/chartoptions/rangeselector/RangeSelector.java

```diff
@@ -0,0 +1,218 @@
+package us.ascendtech.highcharts.client.chartoptions.rangeselector;
+
+import elemental2.core.JsArray;
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+import us.ascendtech.highcharts.client.chartoptions.rangeselector.Buttons;
+
+
+/**
+ * @author Samuel Palacean
+ * Docs: https://api.highcharts.com/highstock/rangeSelector
+ */
+@JsType(isNative = true, name = "Object", namespace = JsPackage.GLOBAL)
+public class RangeSelector {
+    private Boolean allButtonsEnabled;
+    private double buttonSpacing;
+    private String buttonTheme;
+    private String dropdown;
+    private Boolean enabled;
```

Comment:
why not primitive type `boolean`?
Are you also handling null values?

Reply:
![image](https://github.com/veracityidinc/gwt-highcharts/assets/11633371/5cfd28bd-5fd7-4a65-a154-4721da90c6e5)


based on documentation there is also undefined value possible. That`s why Boolean.

Reply:
got it 👍🏼